### DISCUSSION TITLE
How do I set the background color of the map to be transparent?

### DISCUSSION THREAD

**Discussion by jayhuang861019** [mapsforge/mapsforge#1476] [Q&A] [answered]
I am developing my own app using osmdroid, and I have encountered a problem. I have a map file that contains only contour lines, and I need to overlay it on top of a satellite layer. However, I found that I cannot set the contour line layer to be transparent. It always has a color, even if I set map-background="#00000000" in the theme file. How can I set it up?

**Comment by ludwigb** [comment, marked as answer]
Are you setting the Layer to be transparent? There is a flag in the
constructor.

https://github.com/mapsforge/mapsforge/blob/ff5ecde0478e03dbfcc0be3934271e70317a239d/mapsforge-map/src/main/java/org/mapsforge/map/layer/renderer/TileRendererLayer.java#L74C5-L78C6

**Comment by jayhuang861019** [comment]
THANKX I will have a try

**Comment by devemux86** [comment]
Also the map themes have some parameters with the background color:
`map-background`, `map-background-outside`
